.gitattributes

```diff
@@ -22,23 +22,27 @@ Lib/test/sndhdrdata/sndhdr.* binary
 PC/classicAppCompat.* binary
 
 # Text files that should not be subject to eol conversion
-Lib/test/cjkencodings/* -text
-Lib/test/decimaltestdata/*.decTest -text
-Lib/test/test_email/data/*.txt -text
-Lib/test/xmltestdata/* -text
-Lib/test/coding20731.py -text
-Lib/test/test_importlib/data01/* -text
-Lib/test/test_importlib/namespacedata01/* -text
+[attr]noeol -text
+
+Lib/test/cjkencodings/*                    noeol
+Lib/test/coding20731.py                    noeol
+Lib/test/decimaltestdata/*.decTest         noeol
+Lib/test/test_email/data/*.txt             noeol
+Lib/test/test_importlib/data01/*           noeol
+Lib/test/test_importlib/namespacedata01/*  noeol
+Lib/test/xmltestdata/*                     noeol
 
 # CRLF files
-*.bat text eol=crlf
-*.ps1 text eol=crlf
-*.sln text eol=crlf
-*.vcxproj* text eol=crlf
-*.props text eol=crlf
-*.proj text eol=crlf
-PCbuild/readme.txt text eol=crlf
-PC/readme.txt text eol=crlf
+[attr]crlf text eol=crlf
```

Comment:
I'm a little wary of this one; IIRC `crlf` is (was?) a built-in attribute, so I'm not entirely sure this will work as we expect, particularly in older versions of `git`.  How thoroughly have you tested/researched this? :)

If you need a test case, you could add `Tools/msi/README.txt` as a crlf file.

Otherwise, this change LGTM.

Reply:
But let us just use a different name (or drop this change) :)